var/spack/repos/builtin/packages/qgraf/package.py

```diff
@@ -20,4 +20,6 @@ class Qgraf(Package):
     def install(self, spec, prefix):
         fortran = Executable(spack_fc)
         fortran("qgraf-{0}.f".format(self.spec.version), "-o", "qgraf")
-        install_tree(".", prefix)
+        # TODO: use share... subdirectories
```

Comment:
I am not sure I understand the TODO here. Should this be installed into `<prefix>/share`?